e2e/sync_test.go

```diff
@@ -1793,3 +1795,44 @@ func TestIssue435(t *testing.T) {
 		assertError(t, err, errS3NoSuchKey)
 	}
 }
+
+// If source path contains a special file it should not be synced
+func TestSyncSocketDestinationEmpty(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	workdir := fs.NewDir(t, t.Name())
+	defer workdir.Remove()
+
+	sockaddr := workdir.Join("/s5cmd.sock")
+	ln, err := net.Listen("unix", sockaddr)
+	if err != nil {
+		t.Fatalf("error occurred while creating a socket file: %v\n", err)
```

Comment:
No need to add new line with `t.Fatal/Fatalf`.
Also you may just `t.Fatal(err)` without context as `go test` would report where it occurred.

Reply:
Resolved. https://github.com/peak/s5cmd/pull/618/commits/c8c642712c1d4b9cbb5daeb2bbfcd3ec5ce39e25